feature-libs/tracking/tms/aep/aep-collector.service.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { TestBed } from '@angular/core/testing';
+import { LoginEvent } from '@spartacus/core';
+import { TmsCollectorConfig, WindowObject } from '@spartacus/tracking/tms/core';
+import { AepCollectorService } from './aep-collector.service';
+
+const config: TmsCollectorConfig = {};
+describe('AepCollectorService', () => {
+  let service: AepCollectorService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(AepCollectorService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  describe('init', () => {
+    it('should initialize the data layer properly', () => {
+      const windowObject: WindowObject = {
+        ...window,
```

Comment:
I would definitely not touch the global window in unit tests (and pass it to the tested function, so it can freely mess with it).